### DISCUSSION TITLE
Switch of charging completly

### DISCUSSION THREAD

**Discussion by approx3145** [wlcrs/huawei_solar#783] [General]
I'd like to switch of the charging in the morning hours completly. I set the maximum charging power to 0. But it still charges. the battery_state_of_capacity is increasing. 

is it possible to switch off charging completly?

**Comment by Roving-Ronin** [comment]
Suggest you look in the FusionSolar portal / app and look in the 'Time of Use' settings, to define if/when it will charge, and also look at from where it will charge.

Also set working mode of batter to 'Maximise Self Consumption', so when there's PV energy it will be used in the house, then battery charging and if anything left then export.

**Comment by algopt** [comment]
I have the same issue, I would like to find how to set hours in HA where the house power came straight from the grid and the battery pack would not charge/discharge.

The other day I left both the battery pack and the car charging from 2 am to 5 am. As the Huawei Lina pack finished charging around 4, for one hour the car charged from the battery pack, in the morning the battery pack was almost drained.
